airflow_clickhouse_plugin/sensors/clickhouse_sql_sensor.py

```diff
@@ -0,0 +1,91 @@
+from typing import Dict, Callable, Any, Optional
+
+from airflow.exceptions import AirflowException
+from airflow.sensors.sql_sensor import SqlSensor
```

Comment:
`SqlSensor` is unused and can be deleted

Reply:
Not valid anymore but thanks for attention